app/main.ipynb

```diff
@@ -25,18 +25,22 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "import numpy\n",
```

Comment:
The import statement `import numpy` is redundant since you are already importing `numpy` as `np`. You can remove this line to avoid unnecessary imports.